src/auth/auth.js

```diff
@@ -67,8 +65,11 @@ function filterGroups(prefix, groups) {
 	return groups.filter(grp => grp.startsWith(prefix)).map(grp => grp.substring(prefix.length)).filter(grp => !isNaN(grp))
 }
 
-function Auth(url) {
-	this.url = url
+function Auth(loginUrl, logoutUrl, sessionsUrl) {
+	// reactive indicator for when session is being loaded from the back-end
+	this.loading = Vue.observable({
```

Comment:
I was trying to understand the memory handling on this, as this adds a listener to the object, but how do we actually remove the listener from this.loading? that `__ob__` thing? and wondering does this cause a memory leak, the `this.loading` does not seem to have any `destroy` functions or anything which could be used to remove the listener.

Reply:
I don't think you generally remove the `__ob__` thing aka Observer (implementation [here](https://github.com/vuejs/vue/blob/dev/src/core/observer/index.js)). It should be garbage collected together with the entire `this.loading` object when there are no references / reactive dependencies to it.

I guess that if you no longer need the object to be reactive, you could convert it back to a plain javascript object with something like `this.loading = JSON.parse(JSON.stringify(this.loading))`, but that won't help with existing watchers that depend on it.

Reply:
I wasn't able to find anything related to this either, lets go with this now and fix it later if this will cause issues.